dev/com.ibm.ws.jpa_locking_fat/fat/src/com/ibm/ws/jpa/TestEntityLocking_Web.java

```diff
@@ -0,0 +1,197 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package com.ibm.ws.jpa;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.jboss.shrinkwrap.api.ArchivePath;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.EnterpriseArchive;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.runner.RunWith;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.websphere.simplicity.config.Application;
+import com.ibm.websphere.simplicity.config.ClassloaderElement;
+import com.ibm.websphere.simplicity.config.ConfigElementList;
+import com.ibm.websphere.simplicity.config.ServerConfiguration;
+import com.ibm.ws.jpa.fvt.entitylocking.tests.web.EntityLocking20WebTestServlet;
+
+import componenttest.annotation.Server;
+import componenttest.annotation.SkipForRepeat;
+import componenttest.annotation.TestServlet;
+import componenttest.annotation.TestServlets;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.topology.database.container.DatabaseContainerType;
+import componenttest.topology.database.container.DatabaseContainerUtil;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.PrivHelper;
+
+@RunWith(FATRunner.class)
+@Mode(TestMode.FULL)
+@SkipForRepeat("JPA20_FEATURES")
+public class TestEntityLocking_Web extends JPAFATServletClient {
+    private final static String RESOURCE_ROOT = "test-applications/EntityLocking/";
+    private final static String appFolder = "web";
+    private final static String appName = "EntityLockingWeb";
+    private final static String appNameEar = appName + ".ear";
+
+    private final static Set<String> dropSet = new HashSet<String>();
+    private final static Set<String> createSet = new HashSet<String>();
+    private final static Set<String> populateSet = new HashSet<String>();
+
+    private static long timestart = 0;
+
+    static {
+        dropSet.add("JPA_ENTITYLOCKING_DROP_${dbvendor}.ddl");
+        createSet.add("JPA_ENTITYLOCKING_CREATE_${dbvendor}.ddl");
```

Comment:
Is populateSet missing? I see there is a `dev/com.ibm.ws.jpa_locking_fat/test-applications/EntityLocking/ddl/WEB-INF/classes/JPA_ENTITYLOCKING_POPULATE_DERBY.ddl`. Is this needed to be run?